_data/redirects.yaml

```diff
@@ -2,6 +2,11 @@
 - source: /join
   destination: https://www.guildofstudents.com/studentgroups/societies/css/
 
+# join
```

Comment:
don't get why this is #join but its not the ned of the world

Reply:
That was my fault for copy and pasting liberally - been corrected